src/room_member.cpp

```diff
@@ -230,12 +230,16 @@ void RoomMember::ReceiveLoop() {
                 break;
             }
 
-            peer->DeallocatePacket(packet);
+            if(IsConnected())           //Otherwise they allready got deallocated by the Shutdown
```

Comment:
Not a good solution imo.
Our `IsConnected` is not directly linked to raknets state. We should simply avoid shutting down while processing packets like I suggested in https://github.com/B3n30/tunnler/issues/30#issuecomment-291908582